packages/ptokens-helpers/src/string/index.ts

```diff
@@ -6,8 +6,8 @@ export function isHexPrefixed(_string: string) {
   return _string.slice(0, 2).toLocaleLowerCase() === HEX_PREFIX
 }
 
-export function addHexPrefix(_string: string) {
-  return isHexPrefixed(_string) ? _string : HEX_PREFIX + _string
+export function addHexPrefix(_string: string): `0x${string}` {
```

Comment:
I don't understand this change. The function has already been tested and working.

Reply:
viem requires  `0x${string}` instead of string where an hash is passed.